lte/gateway/c/core/oai/tasks/ngap/ngap_amf_nas_procedures.c

```diff
@@ -59,6 +59,9 @@
 #include "ngap_amf_handlers.h"
 #include "ngap_common.h"
 
+int ngap_fill_pdu_session_resource_setup_request_transfer(
+    const pdu_session_resource_setup_request_transfer_t* session_transfer,
+    Ngap_PDUSessionResourceSetupRequestTransfer_t* transfer_request);
```

Comment:
seems you could use the declaration in lte/gateway/c/core/oai/tasks/ngap/ngap_common.h ?

Reply:
Yes, will make the change